mii/config.py

```diff
@@ -18,7 +18,7 @@
 class ReplicaConfig(DeepSpeedConfigModel):
     hostname: str = ""
     tensor_parallel_ports: List[int] = []
-    torch_dist_port: int = None
+    torch_dist_port: int = 0
```

Comment:
I'm not sure if this will have downstream effects. I need to take a closer look at how the DeepSpeed launcher behavior will be changing this from `None` to `0`. Perhaps we need to set it as a reasonable default or specify it as `torch_dist_port: Optional[int] = None`

Reply:
Good point. I'll take a look as well

Reply:
I reverted the torch_dist_port change back to None.

Reply:
Thanks @blackmambaza - on closer inspection, I realize the default port is set in `ModelConfig` correctly:
https://github.com/microsoft/DeepSpeed-MII/blob/ab6b544fcff392fe9004b8832d673d037ede2d87/mii/config.py#L77

This value gets propagated to the `ReplicaConfig`. Since `ReplicaConfig` is not intended to be used directly by users, I think it's ok to have `None` as the default value:
https://github.com/microsoft/DeepSpeed-MII/blob/ab6b544fcff392fe9004b8832d673d037ede2d87/mii/config.py#L358